distuv/norm_example_test.go

```diff
@@ -0,0 +1,39 @@
+// Copyright ©2017 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package distuv_test
+
+import (
+	"fmt"
+	"math"
+
+	"github.com/gonum/stat"
+	"github.com/gonum/stat/distuv"
+)
+
+func ExampleNormal() {
+	// Create a normal distribution
+	dist := distuv.Normal{
+		Mu:    2,
+		Sigma: 5,
+	}
+
+	data := make([]float64, 100000)
+
+	// Draw some random values from the standard normal distribution
+	for i := range data {
+		data[i] = dist.Rand()
+	}
+
+	mean, std := stat.MeanStdDev(data, nil)
+	meanErr := stat.StdErr(std, float64(len(data)))
+	stdErr := std * math.Sqrt(2/(float64(len(data))-1))
```

Comment:
I think what you wrote is the standard error in the variance. The error in the standard deviation would be (roughly) square root that, so 

````
std * math.Sqrt(math.Sqrt(2/float64(len(data)-1))
````

I'm not sure what we should do about `std` vs. `dist.Sigma`. Mathematically, it's only accurate with `dist.Sigma`, though if you know `dist.Sigma` there's no point in computing an error in the standard deviation. Similarly for the mean, one does not normally know the standard deviation, and so one estimates it with data (as is done here now). We may want to make a comment to that effect though.